Automattic/wp-calypso
# Hosting onboarding: allow importing a site without specifying a target WPCOM site

Closes https://github.com/Automattic/dotcom-forge/issues/2585.

## Proposed Changes

Let's add a new `/setup/import-hosted-site` so that developers can import sites without a target site already established in WPCOM.

[We're not adding the data center picker and plugins/themes description right now](https://github.com/Automattic/wp-calypso/pull/77588#issuecomment-1573842072).

## Testing Instructions

1. Enable `hosting-onboarding-i2`;
2. Make yourself a developer by toggling "I am a developer" in `/me`;
3. Browse Sites > Import existing site: it should land you on `/setup/import-hosted-site`;
4. Pass a Jurassic Ninja site.

The import flow should work as normal. You should be asked to upgrade near the end. Upgrade and pay for Business with free credits.

The site on WPCOM should have its URL derived from the source site's title;

You should see a CTA to go to the site's home.
